bindings_flutter/src/libxmtp_api.rs

```diff
@@ -62,25 +63,41 @@ pub fn user_preferences_decrypt(
     .map_err(|e| XmtpError::Generic(anyhow::Error::msg(e)))
 }
 
+pub type XmtpClient = MlsClient<ApiClient>;
+
 #[frb(opaque)]
 pub struct Client {
-    // We use this second wrapper to keep the inner client opaque.
-    pub inner: Arc<InnerClient>,
+    pub inner: Arc<XmtpClient>,
```

Comment:
This second wrapper wasn't necessary (so this PR includes a handful of `.inner.client` -> `.inner`).